docs/accessibility/best-practices/tooltip.md

```diff
@@ -0,0 +1,74 @@
+Page Template
+============
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Tooltip
+
+Tooltips are useful elements. They allow presenting information with an icon or a short text and adding additional longer description when the users hovers the icon or text. 
+If serveral tooltips are on a page, every toolpip must have an own ID. Other wise a screenreader cannot identify which tooltip gives Information to which element.
+
+#`<div class=".." tabindex="0" aria-describedby="tip_<?php echo $i; ?>">
+    <span class="icon-eye">
+</div>
+<div role="tooltip" id="tip_<?php echo $i; ?>">
+    <?php echo Text::_('Text to display what icon-eye means in this context'); ?>
+</div>`
```

Comment:
would you like to write an example code block then you have to write 3 \` on it's own line also ending with 3 \` on it's own line
```suggestion
```html
<div class=".." tabindex="0" aria-describedby="tip_<?php echo $i; ?>">
    <span class="icon-eye">
</div>
<div role="tooltip" id="tip_<?php echo $i; ?>">
    <?php echo Text::_('Text to display what icon-eye means in this context'); ?>
</div>
```
```